spec.html

```diff
@@ -7637,13 +7637,11 @@ <h1>Jobs and Host Operations to Enqueue Jobs</h1>
     <p>Jobs are scheduled for execution by ECMAScript host environments. This specification describes the host hook HostEnqueuePromiseJob to schedule one kind of job; host environments may define additional abstract operations which schedule jobs. Such operations accept a Job Abstract Closure as the parameter and schedule it to be performed at some future time. Their implementations must conform to the following requirements:</p>
 
     <ul>
-      <li>At some future point in time, when there is no running execution context and the execution context stack is empty, the implementation must:
+      <li>At some future point in time, when there is no running execution context and the execution context stack is empty, the implementation must perform the following:
         <ol>
-          <li>Push an execution context onto the execution context stack.</li>
           <li>Perform any implementation-defined preparation steps.</li>
```

Comment:
```suggestion
          <li>Any implementation-defined preparation steps.</li>
```

Reply:
I prefer to keep a verb there.

Reply:
"implementations must perform the following: perform" makes no sense. If we want to keep the verb here, then we'd need to change the preamble to the list.

Reply:
It doesn't add more information, I guess, but it makes sense. Do you take similar issue with AOs saying "It performs the following steps when called" and one of the steps might be e.g. "Perform ? DefinePropertyOrThrow(...)"?

Reply:
No, because in that case each step contains its own verb.

This isn't a huge deal obv, i wouldn't block the PR on it, but I felt it reads awkwardly.

Reply:
Each step here also contains its own verb, before your suggested change. I'm confused.

Reply:
I think the double use of "perform" is what reads awkwardly to me. Maybe if the list preamble said "implementations must:", then keeping "perform" here would be fine?

Reply:
I like that suggestion, will change to that.